test/graaflib/graph_test.cpp

```diff
@@ -222,4 +255,28 @@ TYPED_TEST(GraphTest, ConstGetter) {
   ASSERT_NO_THROW(test_getters_on_const_graph(graph));
 }
 
+TEST(GraphTest, Transpose) {
+  // GIVEN
+  using graph_t = directed_graph<int, int>;
+  graph_t graph{};
+  const auto vertex_id_1 = graph.add_vertex(1);
```

Comment:
Same for this test, we can then also move it to its own file

Reply:
No prob, will do!